docusaurus/docs/using-https-in-development.md

```diff
@@ -32,13 +32,21 @@ HTTPS=true npm start
 
 Note that the server will use a self-signed certificate, so your web browser will almost definitely display a warning upon accessing the page.
 
-To avoid having to set the environment variable each time, you can either include in the `npm start` script like so:
+To avoid having to set the environment variable each time, you can include it in the `npm start` script like so:
 
 ```json
 {
   "start": "HTTPS=true react-scripts start"
 }
 ```
 
-Or you can create a `.env` file with `HTTPS=true` set.
+However, this will only work on Linux and macOS (Bash), as Windows defines the environment variables differently. You can use [`cross-env`](https://www.npmjs.com/package/cross-env) to overcome this issue. The package provides a seamless way of defining cross-platform environment variables through the CLI. After installing `cross-env` through your favourite package manager, you can use it to include the environment variable in the `npm start` script like so:
```

Comment:
@thexpand thanks for the PR! How about keeping `### Linux, macOS (Bash)` as-is, and adding a separate header `### Linux, Windows` so that it's more clear at a glance?

cc @oluoluoxenfree

Reply:
@amyrlam Yes, it makes sense. I'll separate it. Maybe the section can be named even better, like this: `### Cross-platform (Linux, Windows, macOS)` - what do you think?

Reply:
@thexpand 👍 sounds good, thank you!